invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/components/common/facets.js

```diff
@@ -0,0 +1,239 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2022 CERN.
+//
+// Invenio App RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import React, { useState } from "react";
+import {
+  Accordion,
+  Button,
+  Card,
+  Checkbox,
+  Label,
+  List,
+} from "semantic-ui-react";
+import Overridable from "react-overridable";
+import PropTypes from "prop-types";
+import { BucketAggregation, Toggle } from "react-searchkit";
+
+export const RDMSearchAppFacets = ({ aggs, toogle }) => {
```

Comment:
All the `RDM...` prefixes in this file should be omitted as there is no rdm context here :)

Reply:
Yes, @kpsherva and I talked about this issue, but we still had left to decide what naming criteria should be used instead to differentiate it from the default components.